MonoGame.Framework/Graphics/Effect/IEffectFog.cs

```diff
@@ -48,32 +10,31 @@ namespace Microsoft.Xna.Framework.Graphics
 	public interface IEffectFog
 	{
         /// <summary>
-        /// The floating point fog color.
+        /// Gets or set the fog color in floating point format.

```

Comment:
> It is useful when you get a read-only property.

I'm fine with calling out a read only property saying "Returns" or "Gets" in those cases.  But why should we repeat the words "gets or sets" on most every other property that isn't read only?

Note that the doc tool and VS intellisense use an icon to show the accessibility of the getter/setter on properties.